README.md

```diff
@@ -0,0 +1,113 @@
+# CreateProcessW
+
+This crate provide an API similar to `std::process` to create and handle process
+using the [windows-rs][windows-rs] crate and the Windows API (see [this example][creating-processes]).
+
+# Usage
+
+Add the following to your `Cargo.toml`:
+
+```toml
+[dependencies]
+CreateProcessW = "0.1.0"
+```
+
+This crate doesn't follow Rust's naming convention, if you want to avoid the
+`non_snake_case` name, you can use the following:
+
+```toml
+[dependencies]
+create_process_w = { version = "0.1.0", package = "CreateProcessW" }
+```
+
+# Create a new Command
+
+To configure and create a new process, you can use the `Command` struct. The
+only argument needed is the command line you want to execute.
+
+```rust
+use CreateProcess::Command;
+
+let command = Command::new("ls");
```

Comment:
ls??? on Windows? :rofl: at least provide a working example

Reply:
Not problem, in this case i receive the files in my crate.
I followed the `std::process` documentation, and it work if i give a `current_dir` too.
`dir` is a better example maybe ?